src/main/scala/cromwell/engine/db/slick/DataAccessController.scala

```diff
@@ -0,0 +1,319 @@
+package cromwell.engine.db.slick
+
+import java.util.{UUID, Date}
+
+import cromwell.binding.{WdlSource, Call}
+import cromwell.binding.types.WdlType
+import cromwell.binding.values.{WdlPrimitive, WdlValue}
+import cromwell.engine.db._
+import cromwell.engine.store.ExecutionStore.ExecutionStatus
+import cromwell.engine.store.SymbolStore.SymbolStoreKey
+import cromwell.engine.store.SymbolStoreEntry
+import cromwell.engine.{WorkflowId, WorkflowState, WorkflowSubmitted}
+import cromwell.util.DatabaseConfig
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ExecutionContext, Await, Future}
+import scala.language.implicitConversions
+
+// TODO: Need lots of refactoring including: better futures handling, compiled queries, utility functions, etc.
+object DataAccessController extends DataAccess {
+  val dataAccess = new DataAccessComponent(DatabaseConfig.slickDriver)
+
+  import dataAccess.driver.api._
+
+  def database: Database = Database.forConfig("", DatabaseConfig.databaseConfig)
+
+  // TODO: Pass in the execution context instead of using the global!!
+  private implicit val executionContext = ExecutionContext.global
+
+  private implicit def date2timestamp(date: java.util.Date): java.sql.Timestamp =
+    new java.sql.Timestamp(date.getTime)
+
+  // WdlValues have multiple ways they coerce back. This currently decoerces for a proper round trip.
+  // TODO: Refactor into the type?
+  private def wdlValueToString(wdlValue: WdlValue) = {
+    wdlValue match {
+      case wdlPrimitive: WdlPrimitive => wdlPrimitive.asString
+      case other => other.toString
+    }
+  }
+
+  /** Stamp the start_dt column with DateTime.Now, WorkflowState as Submitted. */
+  override def createWorkflow(id: WorkflowId, wdlUri: String, wdlSource: WdlSource, jsonInputs: String, symbols: Seq[SymbolStoreEntry]): Unit = {
```

Comment:
@kshakir - I was just talking to @mcovarr in person and unless you've already picked up on this I think there's a big problem here. The ExecutionStore creates a Call in NotStarted state for every Call in the workflow (analogous to an entry in the execution table). I think what needs to happen here is something similar to `symbols` and take in the calls to populate. 

This should be easy to patch up although we should make sure the tests have an assertion that this is happening before proceeding.